content/comm/conferences/index.en.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "R-Ladies Conference Participation"
+menuTitle: "Conference Participation"
+weight: 110
+---
+
+R-Ladies participates in conference sessions such as contributed talks,
```

Comment:
Are there any criteria for the conferences? Does it have to be conferences around only R/Statistic?

Reply:
Good question. I'm not sure. I can imagine there might be some conferences outside the scope of R and statistics which may interested in hearing from the org.